book/src/accounts/account_policy.md

```diff
@@ -188,6 +188,17 @@ account policy for a group. For example, to set the allowlist for all persons, r
 kanidm group account-policy webauthn-attestation-ca-list idm_all_persons trusted-authenticators
 ```
 
+### Setting Primary Credential Fallback
+
+The primary credential fallback enables behavior which allows authenticating
+using the primary account password when logging in via LDAP.
+
+If both a LDAP and primary password are specified it will only accept the LDAP password.
+
+```bash
```

Comment:
Is there "disable" option? How does one complete that step?

Reply:
Default is set to false. If its enabled and it should be disabled, just running:
`kanidm group account-policy allow-primary-cred-fallback <group name> false` will work.

Reply:
No stress, can do

Reply:
@yaleman sorry for the delay. (Got very busy with work). This good?

Reply:
No worries, I'm on holiday and this is a fun thing not super urgent 😃 thanks for the update